addons/website_event/models/event.py

```diff
@@ -78,14 +77,12 @@ def _set_show_menu(self):
     def _get_show_menu(self):
         self.show_menu = bool(self.menu_id)
 
-    def google_map_img(self, cr, uid, ids, zoom=8, width=298, height=298, context=None):
-        event = self.browse(cr, uid, ids[0], context=context)
-        if event.address_id:
-            return self.browse(cr, SUPERUSER_ID, ids[0], context=context).address_id.google_map_img()
+    def google_map_img(self, zoom=8, width=298, height=298):
+        if self.address_id:
```

Comment:
There won't be a case when we will have multiple ids for this method, but for the sake of safety, added ensure_one.